ionic2/app/services/api.service.ts

```diff
@@ -74,6 +74,16 @@ export class ApiService {
   }
 
   /**
+   * Get Pokemon by description.
+   * @param {string} description - Description of the Pokemon
+   * @returns {Observable<Pokemon[]>}
+   */
+  getPokemonByDescription(description: string): Observable<Pokemon[]> {
```

Comment:
@WoH How do you mean that?
The PokeData team has all those changes in a pull request atm :)